instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/messaging/MessageOperation.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.api.instrumenter.messaging;
+
+/**
+ * Represents type of <a
+ * href="https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/messaging.md#operation-names">operations</a>
+ * that may be used in a messaging system.
+ */
+public enum MessageOperation {
+  SEND,
+  RECEIVE,
+  PROCESS;
```

Comment:
I will ask my favourite question: why enum cannot be already in lower case?

Reply:
Are these missing from the generated conventions? If so let's fix it.

Until then we could also go with static final String which is what our generator produces since yesterday

Reply:
The semconv generated enum is missing the `send` value:

```java
  public enum MessagingOperationValues {
    /** receive. */
    RECEIVE("receive"),
    /** process. */
    PROCESS("process"),
    ;
  }
```

I suppose that's because setting `message.operation` to `send` is strictly forbidden in the spec. Still, having `send` available makes the instrumentation code a bit clearer...

> why enum cannot be already in lower case?

Good question - I don't really mind it, but most people tend to dislike anything other than CONSTANT_CASE.